### PR TITLE
Blog post: Live Monitoring With Shoov And Some Sweet Irony

### PR DESCRIPTION
**Pull request by amitaibu**

### CODE REVIEW

**Comment by amitaibu** [comment]
@niryariv  ready for review

**Comment by amitaibu** [comment]
Merged and deployed -- http://www.gizra.com/content/live-monitor-shoov-irony/

File: app/_posts/2015-06-01-shoov-post-mortem.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+Content-Length: 111
+
+client_id=007xxxx007&amp;client_secret=1xxxxxxa2&amp;code=someOneTimeCode
+```
+
+As you might have noticed (and I can't blame you if you didn't) the `&` char was escaped, and GitHub decided it doesn't like that anymore.
+
+## Solve and Prevent Regressions
+
+After quickly solving the error by hardcoding the `&` char I've decided to spend some time in figuring how I could prevent this from happening again. (Remember: Shoov means "again" in Heberw for this very reason...)
```
Comment (bricel) [inline app/_posts/2015-06-01-shoov-post-mortem.md]: Heberw => Hebrew

Comment (amitaibu) [inline app/_posts/2015-06-01-shoov-post-mortem.md]: fixed

**Pull request closed without merging**